### DISCUSSION TITLE
Can't access live audio

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#515] [Q&A]
Hi,

I can access live audio from my http://birdnet.local, however I can't from my domain. When I click live audio there is a pop up asking for a username (which is birdnet) and a password (which I set in the advanced settings) 

> BirdNET-Pi Password
> This password will protect your "Tools" page and "Live Audio" stream

What can be wrong?

Thank you.

**Comment by blogdemoi** [comment]
And what should be the username? pi or birdnet ? When I ssh in my Birdnet-Pi and do `cat /etc/passwd | grep home` which should return all the users with home directories I have only pi.

**Reply by ehpersonal38** [reply]
It's birdnet. The live audio or any protected page [checks](https://github.com/mcguirepr89/BirdNET-Pi/blob/main/homepage/index.php):
```
// get CADDY_PWD from /home/pi/BirdNET-Pi/birdnet.conf
$caddypwd = $config['CADDY_PWD'];

if($submittedpwd == $caddypwd && $submitteduser == 'birdnet'){
 // access granted
}
```

Really not sure why it doesn't work on public but does on local. I haven't seen this bug report anywhere else so I can only assume your web server is misconfigured somehow.

**Reply by mcguirepr89** [reply]
https://github.com/mcguirepr89/BirdNET-Pi/discussions/508#discussioncomment-3435789
